srv/components/misc_pkgs/lustre/config.sls

```diff
@@ -24,7 +24,11 @@ Update lnet config file:
 {%- if 'data0' in grains['ip4_interfaces'] and grains['ip4_interfaces']['data0'] %}
   {%- set data_if = 'data0' -%}
 {% else %}
+  {%- if pillar['cluster'][node]['network']['data_nw']['iface']|length > 1 %}
```

Comment:
variable node is undefined in this file. Please use grains['id'] or define node variable before.